src/components/rangeslider/range_plot.js

```diff
@@ -62,7 +63,9 @@ module.exports = function rangePlot(gd, w, h) {
             var posX = w * (x[k] - minX) / (maxX - minX),
                 posY = h * (1 - (y[k] - minY) / (maxY - minY));
 
-            pointPairs.push([posX, posY]);
+            if(!isNaN(posX) && !isNaN(posY)) {
```

Comment:
This will be relevant for #472. For the moment, this will guard for thrown errors when passing the `pointPairs` to `makeScatter`.